MAINTAINERS.yml

```
@@ -259,6 +259,20 @@ Ambiq Platforms:
   labels:
     - "platform: Ambiq"
 
+BeagleBoard Platforms:
+  status: maintained
+  maintainers:
+    - jadonk
+  collaborators:
+    - ayush1325
+    - vaishnavachath
+    - con-pax
+  files:
+    - boards/arm/beagle_bcf/
+    - boards/riscv/beaglev_fire/
```

probably doesn't hurt to do it this way:

```suggestion
    - boards/arm/beagle*
    - boards/riscv/beagle*
```